bot/common/commands.py

```diff
@@ -341,7 +341,7 @@ async def add_dao(ctx: discord.ApplicationContext):
 #                return
 #            thread = await GuildSelect(
 #                ctx.author.id,
-#                hashlib.sha256("".encode()).hexdigest(),
+#                None,
 #                message.id,
 #                "",
 #            )
```

Comment:
We can probably remove this commented out step right?

Reply:
yep we def can.